vraygolaem.cpp

```diff
@@ -884,6 +887,12 @@ void VRayGolaem::updateVRayParams(TimeValue t)
 
 	// vray
 	_frameOffset = pblock2->GetInt(pb_frame_offset, t);
+	const TCHAR *defaultMat_wstr=pblock2->GetStr(pb_default_material, t);
+	if (!defaultMat_wstr) _vrsceneFile="";
```

Comment:
Oups, let me fix this right now

---
[image: Nicolas Chaverou - Golaem Crowd Product Manager - Golaem SA]

+33 (0)2 99 27 21 44
http://www.golaem.com

On Tue, Oct 20, 2015 at 10:59 PM, vkoylazov <notifications@github.com>
wrote:

> In vraygolaem.cpp
> <https://github.com/ChaosGroup/vraygolaem_max/pull/6#discussion_r42553401>
> :
>
> > @@ -884,6 +887,12 @@ void VRayGolaem::updateVRayParams(TimeValue t)
> >
> >  	// vray
> >  	_frameOffset = pblock2->GetInt(pb_frame_offset, t);
> > +	const TCHAR *defaultMat_wstr=pblock2->GetStr(pb_default_material, t);
> > +	if (!defaultMat_wstr) _vrsceneFile="";
>
> Why do you set the _vrsceneFile member to "" here? Should it not be
> _defaultMaterial?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ChaosGroup/vraygolaem_max/pull/6/files#r42553401>.
>